crates/miden-lib/asm/miden/account.masm

```diff
@@ -288,7 +288,10 @@ export.get_code_commitment
     # => [CODE_COMMITMENT]
 end
 
-#! Gets the storage commitment of the account the transaction is being executed against.
+#! Gets the account storage commitment of the current account.
```

Comment:
I wonder if there is value in having this procedure be a part of the public API. The way it is right now, there is no good way to tell if this is returning the initial commitment to account storage or the current commitment to account storage. It may be better to split this into 2 procedures:

- `get_init_storage_commitment`.
- `compute_current_storage_commitment`.

And for consistency, we should probably do the same with account code (even though we can't mutate it yet):

- `get_init_code_commitment`.
- `compute_current_code_commitment`.

And finally, would probably good to add `get_init_vault_root` and rename `get_vault_root` into `get_current_vault_root`.

Since these are somewhat bigger changes, we could do them in a separate PR.

Reply:
Agreed, I thought about this as well. I wasn't sure if we have a guideline around "number of kernel procedures" and if there is a limit that we should not exceed.

In any case, this should be a follow-up PR so I created https://github.com/0xMiden/miden-base/issues/1537.